pysatSeasons/avg.py

```diff
@@ -23,77 +25,74 @@ def median1D(const, bin1, label1, data_label, auto_bin=True, returnData=False):
     binX: array-like
         List holding [min, max, number of bins] or array-like containing
         bin edges, where X = 1, 2
-    labelX: string
+    labelX: str
         identifies data product for binX, where X = 1, 2
     data_label: list-like
         contains strings identifying data product(s) to be averaged
-    auto_bin: boolean
+    auto_bin: bool
         if True, function will create bins from the min, max and
         number of bins. If false, bin edges must be manually entered
-    returnData : boolean
+    returnData : bool
         Return data in output dictionary as well as statistics
 
     Returns
     -------
-    median : dictionary
+    median : dict
         1D median accessed by data_label as a function of label1
         over the season delineated by bounds of passed instrument objects.
         Also includes 'count' and 'avg_abs_dev' as well as the values of
         the bin edges in 'bin_x'
 
     """
 
-    # const is either an Instrument or a Constellation, and we want to
-    #  iterate over it.
-    # If it's a Constellation, then we can do that as is, but if it's
-    #  an Instrument, we just have to put that Instrument into something
-    #  that will yeild that Instrument, like a list.
+    # `const` is either an Instrument or a Constellation, and we want to
+    #  iterate over it. If it's a Constellation, then we can do that as is,
+    #  but if it's an Instrument, we just have to put that Instrument
+    #  into something that will yield that Instrument, like a list.
     if isinstance(const, pysat.Instrument):
         const = [const]
     elif not isinstance(const, pysat.Constellation):
         raise ValueError("Parameter must be an Instrument or a Constellation.")
 
-    # create bins
-    # seems to create the boundaries used for sorting into bins
+    # Create the boundaries used for sorting into bins
     if auto_bin:
         binx = np.linspace(bin1[0], bin1[1], bin1[2] + 1)
     else:
         binx = np.array(bin1)
 
-    # how many bins are used
+    # How many bins are used
     numx = len(binx) - 1
-    # how many different data products
+    # How many different data products
```

Comment:
```suggestion

    # How many different data products
```

Reply:
Have a whitespace before new comments that have the same indentation level.  Makes things easier to read.